t/batches.t

```diff
@@ -310,8 +311,10 @@ subtest 'duplicate user should fail' => sub {
     my $config = Zonemaster::Backend::Config->parse( $config );
     my $backend = init_from_config( $db_backend, $config );
 
-    dies_ok {
-        $backend->add_api_user( { username => $user->{username}, api_key => "another api key" } );
+    capture {
```

Comment:
As an alternative to this, have you tried using [PrintWarn](https://metacpan.org/pod/DBI#PrintWarn) or one of the other techniques suggested there?

Reply:
Nope I didn't tried, but this looks way better. I've provided an update.